integration-tests/src/test/java/oracle/weblogic/kubernetes/ItServerStartPolicy.java

```diff
@@ -565,7 +565,7 @@ void testConfiguredServerLifeCycleWithoutAdmin() {
 
       // Check if pod in init state
       // Here the server pd is created but does not goes into 1/1 state
-      checkPodInitializing(serverPodName, domainUid, domainNamespace);
+      checkPodInitialized(serverPodName, domainUid, domainNamespace);
```

Comment:
Can we add a wait/retry condition till the pod is initialized instead of returning immediately.

Reply:
this is how that method is implemented - it has testUntil

Reply:
Don't you think, this may lead to intermittent test failure ? Can we implement a solution to work around this.